plugins/scaffolder-backend/src/lib/templating/filters/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ScmIntegrations } from '@backstage/integration';
+import parseRepoUrl from './parseRepoUrl';
+import parseEntityRef from './parseEntityRef';
+import pick from './pick';
+import projectSlug from './projectSlug';
+
+export default (options: { integrations: ScmIntegrations }) => [
```

Comment:
I know this was there before but thinking that this should be `export const DEFAULT_TEMPLATE_FILTERS` or something instead.

Reply:
TBH I changed to the default export in the previous PR: [commit.](https://github.com/backstage/backstage/pull/28813/files#diff-5054438e7b756c457a434ed56b1ce80e86e959a64774b169fef1b1109804750c). Shall I just change this back to that, and avoid default exports in future?

Reply:
Restored to original name and "reachable" position, i.e. `filters/index.ts` exports `createDefaultFilters`. Resolving this conversation, leaving the above one on the actual filename open.